README.md

```diff
@@ -34,7 +34,6 @@ communicating with Mojang. Furthermore, this project is used for experimenting w
 * Java 8+
 * Gson (2.2.4+)
 * Guava (10.0.1+)
-
 ## Contribution
 
```

Comment:
I gave a second chance to Cupid
I believed you, I'm so stupid
Let me show you my secret love, is it real?
Cupid is so dumb